Gemfile.lock

```diff
@@ -222,6 +238,7 @@ DEPENDENCIES
   listen (>= 3.0.5, < 3.2)
   puma (~> 3.7)
   rails (~> 5.1.3)
+  rubocop
```

Comment:
👍 

```
$ rake test
...
15 files inspected, 30 offenses detected
```
😢 

You can either address these now or create an action item if you feel it's too much creep for this story.